static/app/views/explore/components/schemaHints/schemaHintsList.tsx

```diff
@@ -241,7 +242,7 @@ function SchemaHintsList({
   useEffect(() => {
     const adjustSearchBarWidth = () => {
       if (isDrawerOpen && searchBarWrapperRef.current && panelRef.current) {
-        searchBarWrapperRef.current.style.width = `calc(100% - ${panelRef.current.clientWidth}px)`;
+        searchBarWrapperRef.current.style.width = `calc(100% - ${source === SchemaHintsSources.LOGS ? panelRef.current.clientWidth - LOGS_QUERY_BUILDER_WIDTH_OFFSET : panelRef.current.clientWidth}px)`;
```

Comment:
It might be better if we can pass this offset down as a parameter so logs or any other caller can define it for themselves and you don't need to have this logic checking the source type